packages/@adobe/spectrum-css-temp/components/table/index.css

```diff
@@ -156,8 +156,12 @@ governing permissions and limitations under the License.
   text-overflow: ellipsis;
 
   /* allow focus ring of child to extend outside the bounds */
-  padding: 0 4px;
-  margin: 0 -4px;
+  padding: 4px;
+  margin: -4px;
+}
+
+.spectrum-Table-cell--hideHeader {
+  padding: 0 2px;
```

Comment:
can you add a comment to explain this one and why it's different from the other one?

Reply:
So looks like the content is not center aligned and this padding helps to align the content 
I can also fix the issue of left focus ring getting cut off with 

```
.spectrum-Table-cell--hideHeader {
   padding: 0; // default padding for other cells is around 16px, so we need to override that
  justify-content: center;
}
```

The same happens to all the buttons in the table when we remove the padding around them and also remove the alignment. checkout the CRUD example
Need your inputs on which method would be better? existing OR above, considering the other pieces in the table

Reply:
setting padding to 0 and justifying center should be the better way to do it i think

Reply:
Agreed, we should be good to go now